src/SFA.DAS.Charities.Import.Functions/Infrastructure/IDateTimeProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SFA.DAS.Charities.Import.Functions.Infrastructure
+{
+    public interface IDateTimeProvider
+    {
+        DateTime Today { get;  }
+    }
+
+    internal class DateTimeProvider : IDateTimeProvider
+    {
+        public DateTime Today { get; } = DateTime.Today;
```

Comment:
Usually with this implementation you have constructor that takes in a DateTime object so that Dependency Injection is used correctly and the DateTime instance is set to the appropriate Globalization settings